### PR TITLE
remove duplicate lines and volumemount keys

### PR DESCRIPTION
**Pull request by mtb-xt**
Fixes issue [#6676](https://github.com/firefly-iii/firefly-iii/issues/6676)

Changes in this pull request:

- removed seemingly duplicated lines, iterating over `additionalVolumeMounts`
- removed second `volumeMounts` key - they all can be covered by the first one.

After these changes, template seems to render correctly, both if I have `additionalVolumeMounts` set or not

### CODE REVIEW

**Review by morremeyer — CHANGES_REQUESTED** [review]
Thank you! This is indeed an issue we have not noticed so far.

Can you please bump the Chart version in `Chart.yaml` by one patch version? Once that's done, re-request my review please.

**Comment by morremeyer** [comment]
@JC5 I was automatically requested as reviewer for this PR, so giving me write access seems to have been successful.

**Comment by mtb-xt** [comment]
@morremeyer done! sorry, wasn't sure about the `stack` chart, so I bumped the dependency to `importer` there too. Or there's an automation for that?

**Comment by morremeyer** [comment]
@mtb-xt The `stack` chart needs the importer to be released first, so that has to be done in a second step. You can open a second PR once the [releases page](https://github.com/firefly-iii/kubernetes/releases) has the importer-1.3.1 release.

I'm planning to automate this soon, but it isn't yet done.

**Comment by JC5** [comment]
I saw that @morremeyer, nice!

You can merge this too right, when it's ready?

**Comment by mtb-xt** [comment]
@morremeyer ok, got it, bumped only this chart!

**Pull request merged**
